config/admin_ranks.txt

```diff
@@ -11,7 +11,9 @@
 ##############################################################################################################
 # PLEASE NOTE: depending on config options, some abilities will be unavailable regardless if you have permission to use them!
 
-# KEYWORDS:
+# Follow the format below when documenting new keywords so the server tools may parse it
+
+# BEGIN_KEYWORDS
```

Comment:
explain why bucko.
all we're doing is defining some valid keywords under a "rank_keywords" key/namespace, and the named ranks in a similar manner.

it does what @Cyberboss wants and still works as a normal config (just needs to the DM side to be changed)